website/docs/cloud-docs/api-docs/organization-tags.mdx

```diff
@@ -40,6 +40,14 @@ description: >-
 
 This API returns the list of tags used with all resources across the organization. Tags can be added to this pool directly or via those resources. Tags deleted here will be removed from all other resources. Tags can be added, applied, removed and deleted in bulk.
 
+Here are some things to note about tags:
```

Comment:
I think my suggestion is still pretty vague, but I can't think of anything better at the moment 😂 :
```suggestion
Tags are subject to the following rules:
```